warehouse/templates/email/api-token-used-in-trusted-publisher-project/subject.txt

```diff
@@ -0,0 +1,21 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+
+{% extends "email/_base/subject.txt" %}
+
+{% block subject %}
+{% trans project_name=project_name %}
+User API token used in project {{ project_name }} with Trusted Publishing enabled
```

Comment:
Extreme nitpick, but we might want to tweak this a bit: this subject could be interpreted as "business as usual," when what we're trying to signal to the user is an unusual case. Maybe this?

```suggestion
Manually configured API token used in project {{ project_name }} despite Trusted Publishing enabled
```

That's a mouthful, though 😅 

(No strong opinion here.)

Reply:
Yeah I agree that we need the subject to convey that there was "maybe unexpected/undesirable" behavior. Looking at the other email subjects in the repo, none have anything explicit like `WARNING: such and such happened`.
I do like your use of `despite`, since it shows that what was observed goes against (usual) expectations.

To reduce verbosity, I think we can get away we removing `Manually configured` since, from the POV of the user, all tokens are manually configured (the short-term tokens minted by the Trusted Publisher process are invisible to the user). So, something like:

`API token used to upload to project {{ project_name }} despite Trusted Publishing enabled`

?

Reply:
SGTM!

Reply:
fixed!